src/vs/workbench/contrib/search/common/searchModel.ts

```diff
@@ -406,6 +409,19 @@ export class FileMatch extends Disposable implements IFileMatch {
 		}
 	}
 
+	async resolveFileStat(fileService: IFileService): Promise<void> {
+		this._fileStat = await fileService.resolve(this.resource, { resolveMetadata: true });
+		return Promise.resolve();
```

Comment:
You don't need to explictly return a promise in an `async` function, this line can be removed.